demo/index.html

```diff
@@ -206,29 +206,33 @@
         </div>
       </template>
       <script>
-        Polymer({
-          is: 'demo-page',
-
-          preload: function(e) {
-            var img = document.querySelector('#' + e.target.getAttribute('target'));
-            img.src = './polymer.svg?' + Math.random();
-            e.target.textContent = 'Reload image';
-          },
-
-          _computeLogoAltText: function(loading, loaded) {
-            var text = 'The Polymer logo. ';
-
-            if (loading) {
-              text += '(loading)';
-            } else if (loaded) {
-              text += '(loaded)';
-            } else {
-              text += '(not loaded)';
-            }
 
-            return text;
-          }
+        window.addEventListener('WebComponentsReady', function() {
```

Comment:
[This page](http://webcomponents.org/polyfills/html-imports/) tries to explain, but it's still pretty unclear. ['HTMLImportsLoaded' === `HTMLImports.whenReady` at least.](https://github.com/webcomponents/webcomponentsjs/blob/07f2169ad902be51054fe6f7d63f3b5a6a2618e9/src/HTMLImports/base.js#L62) The 'WebComponentsReady' event sounds like it's the most safe option though.